internal/services/apimanagement/api_management_api_resource_test.go

```diff
@@ -778,6 +752,42 @@ resource "azurerm_api_management_api" "test" {
 `, r.template(data, SkuNameConsumption), data.RandomInteger)
 }
 
+func (r ApiManagementApiResource) completeUpdate(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_api_management_api" "test" {
+  name                = "acctestapi-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  api_management_name = azurerm_api_management.test.name
+  display_name        = "Butter Parser Update"
+  path                = "butter-parser-update"
+  protocols           = ["https"]
+  revision            = "3"
+  description         = "What is my purpose? You parse butter update."
```

Comment:
Because this is changing every updatable attribute, we'll miss a bug in the current update method. I encourage you to keep this value the same and run this test again to see what happens